firmware/init/sensor/init_tps.cpp

```diff
@@ -125,7 +125,7 @@ struct RedundantPair {
 
 		if (isFordTps && fordTps) {
 			// we have a secondary
-			fordTps->configure(5.0f, 52.6f);
+			fordTps->configure(5.0f, secondaryMaximum);
 			fordTps->Register();
```

Comment:
Should `RedundantPair` and/or `FuncSensPair` need a `deinit` or to otherwise [about here] unregister `m_redund`/`fordTps` in the case the controlling setting is toggled during run, to avoid `Duplicate registration for sensor "..."`?

Reply:
see https://github.com/mck1117/rusefi/pull/4/files for fix